### DISCUSSION TITLE
Eraser view

### DISCUSSION THREAD

**Discussion by connoranderson-synthego** [bordaigorl/rmview#37] [Ideas]
It would be really useful to be able to see the eraser similarly to how we can see the pen tip. Maybe the eraser could be an opaque circle with a larger diameter?

**Comment by bordaigorl** [comment]
There is no reliable way to detect which tool is being used from rmview.
rmview is receiving raw pointer events and the raw pixel data. High level information like the current tool or the orientation are only known to xochitl and there is no signal emitted/API about them that rmview can hook into

**Reply by stheid** [reply]
for the physical eraser of the pro-pen it would be theoretically possible, but than it would also be inconsistent to erasing with the eraser-tool
